app/src/main/java/com/github/multimatum_team/multimatum/QRGenerator.kt

```diff
@@ -0,0 +1,54 @@
+package com.github.multimatum_team.multimatum
+
+import android.content.Intent
+import android.graphics.Bitmap
+import android.graphics.Color
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.EditText
+import android.widget.ImageView
+import android.widget.Toast
+import com.google.zxing.BarcodeFormat
+import com.google.zxing.qrcode.QRCodeWriter
+
+class QRGenerator : AppCompatActivity() {
```

Comment:
Would be nice to have a small docstring explaining what the activity does and briefly describe the interface